docs/adr/0005-remote-settings-client.md

```diff
@@ -0,0 +1,112 @@
+# A remote-settings client for our mobile browsers.
+
+* Status: proposed
+* Deciders:
+    * csadilek for the mobile teams ✔️
+    * bwong for the remote-settings team
+    * mhammond for the application-services team ✔️
+    * [others?]
+* Date: 2022-12-16
+
+## Context and Problem Statement
+
+Mozilla’s mobile browsers have a requirement to access the remote settings service, but currently lack any libraries or tools which are suitable without some work.
+A concrete use case is the management of search engine configurations, which are stored in Remote Settings for Firefox Desktop, but shipped as individual files on our mobile browsers, requiring application releases for all changes.
+This document identifies the requirements, then identifies tools which already exist and are close to being suitable, then identifies all available options we can take, and outlines our decision.
+
+### Requirements
+The requirements are for a library which is able to access Mozilla’s Remote Settings service and return the results to our mobile browsers.
+This list of requirements is not exhaustive, but instead focuses on the requirements which will drive our decision making process.
+As such, it identifies the non-requirements first.
+
+### Not a requirement
```

Comment:
nit: Should the title here be indented i.e a `####` and not a `###`

Reply:
I think the heading should have been `##`! Hopefully addressed in 3c356141e9177b963fbaf31aa07e9196b343cc27